features/step_definitions/deployment.rb

```diff
@@ -62,7 +62,10 @@
   success = wait_for(seconds) do
     rs_name != deplmnt.current_replica_set(user: user, cached: false).name
   end
-  raise 'expected replica set name change not reached within timeout' unless success
+  unless success
+    logger.error(user.cli_exec(:describe, resource_name: "deployment/#{d_name}"))
+    raise "Deployment did not become ready"
```

Comment:
Maybe we need to update the msg to reflect the real status ?

Reply:
@liangxia we could see the deployment status from the output of `oc describe` command .

Reply:
@liangxia Any other suggestion we need to update ?

Reply:
I mean the msg in **raise** is not reflecting the real situation.